### ISSUE TITLE
Show only 10 versions in project view

### ISSUE THREAD

**Issue by zivry** [codeine-cd/codeine#154]
When checking codeine project which has a lot of versions, it takes a lot of time to load, 
maybe need to limit number of versions displayed with more. 
or limit by number of total nodes. 
that will make the page load faster and behave better in browser.

**Comment by ezraroi** [comment]
Agree, also we need to consider implementing the version user control in React as this should improve page rendering time.
